### ISSUE TITLE
Scene based "Maximum Vision Distance " not working but global is.

### ISSUE THREAD

**Issue by Watson-203** [ruipin/fvtt-token-vision-tweaks#3]
As the title says. The "Maxium Vision Distance" setting on a scene doesn't appear to be working although the global option is. Can I give any more info to help troubleshoot?

**Comment by ruipin** [comment]
Yeah, this is actually a known bug after the update to 0.7.5 which I didn't have time to fix back then and forgot to write down. Thanks for opening a ticket, as I don't use this setting I had basically forgotten about this.

Unfortunately I've been busy irl so not sure when I'll have some time to look at this - also been spending a lot of dev time on [libWrapper](https://github.com/ruipin/fvtt-lib-wrapper) - but will try to fix this soon.

**Comment by Watson-203** [comment]
Righto, thanks for getting back to me and confirming it wasn't something on my end. I really appreciate the work you've done with your modules!

**Comment by IchbinkeinReh** [comment]
@Watson-203 is the language of your world different from english? I saw in the code of this module that it tries to match the english heading "Vision". If it does not find that (eg because of a different language) then it does not show the scene settings

**Comment by ruipin** [comment]
Hah, that is actually likely to be it. I didn't think of that. Thanks for noticing!

The vision fixes in 0.8.2 make this module almost obsolete and libWrapper has been consuming most of my Foundry devleopment time, so I don't expect to work any further on this module, though. But you could probably patch that check yourself if not using english.

**Comment by Watson-203** [comment]
My language was/is in English just so you're aware.
